src/Gold.Redis/Gold.Redis.Common/Models/Configuration/RedisConnectionConfiguration.cs

```diff
@@ -1,13 +1,10 @@
-﻿using System;
```

Comment:
please revert this file and commit with smaller commits. i think that the removal of the first lines and the edit of the class itself messed it up. with smaller commits it should not happend